sdk/ai/azure-ai-projects/FunctionTool.md

```diff
@@ -99,3 +107,56 @@ To ensure `FunctionTool` operates correctly and generates accurate function defi
  
    - **Nullable Types**
      - `Optional[type]` includes `null`
+
+2. **Enum enforcement**
+    
+    To use an Enum, the parameter should be of type `str`. You can specify the constraint of the values in the parameter's description. In most cases, OpenAI can understand the description and apply the constraint accordingly. However, the `FunctionTool` class cannot parse the description to generate this constraint in the definition. Therefore, it is recommended to add this constraint programmatically. Here is an example:
```

Comment:
Regarding "OpenAI can understand" - Agents are supposed to work against some non-OpenAI models, if I understand correctly. So I would not use the company name OpenAI in our docs. I would say something like "Most Agent-supported AI models can understand ..."

Reply:
use the word agents instead.